servers/visual/rasterizer.h

```diff
@@ -930,6 +930,7 @@ class Rasterizer {
 	virtual RasterizerScene *get_scene()=0;
 
 	virtual void initialize()=0;
+	virtual void update_system_fbo(uint32_t p_system_fbo)=0;
```

Comment:
I've made this change but I haven't been able to compile and test it yet because of the new audio approach....